recipe-server/normandy/recipes/api/v1/serializers.py

```diff
@@ -205,13 +209,25 @@ class ClientSerializer(serializers.Serializer):
 class SignatureSerializer(serializers.ModelSerializer):
     timestamp = serializers.DateTimeField(read_only=True)
     signature = serializers.ReadOnlyField()
-    x5u = serializers.ReadOnlyField()
+    x5u = serializers.SerializerMethodField()
     public_key = serializers.ReadOnlyField()
 
     class Meta:
         model = Signature
         fields = ['timestamp', 'signature', 'x5u', 'public_key']
 
+    def get_x5u(self, signature):
+        x5u = signature.x5u
+        if x5u is None:
+            return None
+
+        # Add cachebust parameter to x5u URL with the current time
+        url_parts = list(urlparse.urlparse(x5u))
+        query = dict(urlparse.parse_qsl(url_parts[4]))
+        query['cachebust'] = str(int(time.time()))
```

Comment:
This might have the side effect of hammering the content-signature server with a bunch of traffic since the cache-bust changes every second, forcing the CDN to pass traffic through to the app server. We should check with @jvehent first and maybe limit it to changing less often.